server/src/test/java/com/linecorp/centraldogma/server/internal/api/auth/PermissionTest.java

```diff
@@ -98,7 +99,9 @@ public static void cleanup() throws Exception {
         @Override
         protected void configure(ServerBuilder sb) throws Exception {
             final ProjectManager pm = new DefaultProjectManager(
-                    rootDir.newFolder(), ForkJoinPool.commonPool(), NoopMeterRegistry.get(), null);
+                    rootDir.newFolder(), ForkJoinPool.commonPool(), Executors.newSingleThreadExecutor(),
```

Comment:
Probably OK to use `MoreExecutors.directExecutor()`?

Reply:
Sorry, I missed it. 😟